test/core_test.clj

```diff
@@ -1,9 +1,20 @@
 (ns core-test
-  (:require [clojure.test :refer [deftest is testing run-tests]]
+  (:require [clojure.java.io :as io]
+            [clojure.test :refer [deftest is testing run-tests]]
             [bb-excel.core :refer [get-sheets get-sheet-names get-sheet
-                                   get-range 
+                                   get-range
                                    ; get-row get-col get-cells crange
-                                   ]]))
+                                   ]])
+  (:import [java.util.zip ZipFile]))
+
+(deftest zipfile-or-nil-test
+  (let [zipfile-or-nil #'bb-excel.core/zipfile-or-nil]
```

Comment:
I read [here](https://stackoverflow.com/questions/37471253/how-can-i-write-unit-tests-for-private-clojure-functions) about some approaches to test private functions and choose simple one. If you prefer utils approach i can rewrite code.